services/workflows-service/scripts/workflows/kyb-kyc-workflow-definition.ts

```diff
@@ -0,0 +1,280 @@
+import { PrismaClient } from '@prisma/client';
+import { kycEmailSessionDefinition } from './kyc-email-process-example';
+import { env } from '../../src/env';
+
+import { defaultContextSchema, StateTag } from '@ballerine/common';
+import { Type } from '@sinclair/typebox';
+
+export const kybKycWorkflowDefinition = {
```

Comment:
Please compare this with `parentKybWithSessionWorkflowDefinition` in order to see the differences.